hazelcast/src/test/java/com/hazelcast/test/jdbc/MSSQLDatabaseProvider.java

```diff
@@ -29,8 +29,9 @@ public class MSSQLDatabaseProvider implements TestDatabaseProvider {
     @Override
     public String createDatabase(String dbName) {
         container = new MSSQLServerContainer<>("mcr.microsoft.com/mssql/server:" + TEST_MSSQLSERVER_VERSION);
+        container.acceptLicense();
         container.withUrlParam("user", container.getUsername())
-                 .withUrlParam("password", container.getPassword());
+                .withUrlParam("password", container.getPassword());
```

Comment:
nit: you could use the fluent API:
```suggestion
                .withUrlParam("password", container.getPassword())
                .acceptLicense();
```